x/protocolpool/go.mod

```diff
@@ -0,0 +1,159 @@
+module cosmossdk.io/x/protocolpool
+
+go 1.21
+
+require (
+	cosmossdk.io/api v0.7.1
+	cosmossdk.io/core v0.12.0
+	cosmossdk.io/depinject v1.0.0-alpha.4
+	cosmossdk.io/errors v1.0.0
+	cosmossdk.io/log v1.2.1
+	cosmossdk.io/math v1.1.3-rc.1
+	github.com/cometbft/cometbft v0.38.0
+	github.com/cosmos/cosmos-proto v1.0.0-beta.3
+	github.com/cosmos/cosmos-sdk v0.47.5
```

Comment:
As it uses a replaced version, can we put here v0.51, so it breaks better if we remove the replace tag